src/platforms/javascript/common/install/index.mdx

```diff
@@ -20,3 +20,35 @@ notSupported:
 ---
 
 <PageGrid />
+
+## When to use which Installation Method
+
+Depending on the concrete needs of your application, you may wonder which installation method you should use.
+
+### Why to use the Loader Script over NPM
+
+Using the Loader Script is the easiest way to start using Sentry. You only need to add a script tag to your application, and we'll take care of setting everything up correctly for you.
+
+- **Easy to set up**: You only need to add a script tag to your application.
+
+- **Easy to upgrade**: We'll take care of always shipping the latest version of the SDK to you - no action needed on your side.
+
+- **Easy to add additional configuration**: You can still configure Sentry to your liking via `Sentry.onLoad()`.
+
+- **Lazy-loading for errors-only mode**: If you only want to use Sentry for errors, we'll only load the SDK when an error occurs. This means that we can reduce the amount of JavaScript loaded on your page until an error occurs.
```

Comment:
```suggestion
- **Lazy-loading**: If you only want to use Sentry for errors, we'll only load the SDK when an error occurs. This means that we can reduce the amount of JavaScript loaded on your page until an error occurs.
```

Don't know about this one a 100% but it's catchier (ie. if I scan for lazy load my eyes will find it more easily) and we mention that it's for errors only anyhow in the following text and the note.